src/tab.js

```diff
@@ -171,8 +171,14 @@ Object.assign(SideTab, {
     }
     return false;
   },
-  isCloseButtonEvent(e) {
-    return e.target && e.target.classList.contains("tab-close");
+  isCloseButtonEvent(e, checkExplicitOriginal) {
+    if (!e.target) {
+      return;
+    }
+    if (!checkExplicitOriginal) {
+      return e.target.classList.contains("tab-close");
```

Comment:
Just want to confirm, I mean is preference between `if (!e.target) return;` and 
```
if (!e.target) {
  return;
}
```
I see this repository using the latter style, any idea for replacing?